### DISCUSSION TITLE
periodic boundary conditions

### DISCUSSION THREAD

**Discussion by youngung** [neperfepx/FEPX#35] [Q&A] [answered]
Can I impose a periodic boundary condition? I don't see such an example so I wonder if that's possible.

**Comment by rquey** [comment, marked as answer]
No, but this is a work in progress.

**Reply by mkasemer** [reply]
Indeed, it is a goal through the fall to implement periodic boundary conditions. Please check future release notes.

**Reply by youngung** [reply]
I'd be exited to hear the news. I just like the way the software is running with allowing all features available through the command line interface. Also, the documentation is very nicely done. Thanks again for sharing the software on an open domain.

**Reply by TomasManik** [reply]
Hi, I am new to FEPX and would like to apply periodic boundary conditions. I can see that this is a work in progress. What is the status now? I recently realized, that FEPX does not support hex elements. Do I understand this correct, that only tet elements of the second order are supported?  
Thank you very much for you answer!

Tomas Manik, PhD.
Norwegian University of Science and Technology, Trondheim

**Reply by rquey** [reply]
It is still work in progress and should be available in the relatively near future.

Yes, your understanding is correct for the types of elements, only 10-node tets are supported.
